.github/workflows/instance_types.yaml

```
@@ -25,7 +25,10 @@ jobs:
         add-paths: db/fixtures/aws_instance_types.yml
         commit-message: Update AWS instance_types
         branch: update_aws_instance_types
+        author: miq-bot <bot@manageiq.org>
+        committer: miq-bot <bot@manageiq.org>
```

For consistency, elsewhere I've been doing this:

```suggestion
        author: ManageIQ Bot <bot@manageiq.org>
        committer: ManageIQ Bot <bot@manageiq.org>
```

Like here: https://github.com/ManageIQ/miq_bot/blob/master/bin/before_install#L9-L10 and https://github.com/ManageIQ/font-fabulous/blob/master/bin/deploy#L5-L6